### PR TITLE
Upd/k8s

### PR DESCRIPTION
**Pull request by theavege**
improved usability

### CODE REVIEW

**Comment by The-Lum** [comment]
Hello @theavege, 

Don't forget also the remarks on:
- #90

With:
> - Could you change "global absolute github UR"L to "local stdlib" inclusion?



Regards,
Th.

File: k8s/OSS/KubernetesApi.puml
Comment (The-Lum) [inline k8s/OSS/KubernetesApi.puml:1]: Hi,

Maybe you could have left all the single files, and just made `include` in the "all" file, _(to reduce size and redundancy)_ as:

`k8s/OSS/all.puml`:
```plantuml
!include <k8s/OSS/KubernetesApi>
!include <k8s/OSS/KubernetesCcm>
...
!include <k8s/OSS/KubernetesVol>
```

**Review by The-Lum — COMMENTED** [review]
Hi @theavege,

See some remarks on:
- #90
- and here for the `all.puml` file.

_[Out off topic]_
Also see to make a version in SVG sprites... _(But... for a new enhancement...)_

Regards,
Th.

**Comment by theavege** [comment]
@The-Lum , I've done.

**Comment by The-Lum** [comment]
@theavege:
 
Thanks,

But the corrections about locale sdlib are missing...

See remarks on:
- https://github.com/plantuml/plantuml-stdlib/pull/92#issuecomment-1935809286

Regards,
Th.

**Comment by The-Lum** [comment]
@theavege:
- Do you need help making corrections? _(from https://github.com/plantuml/plantuml-stdlib/pull/90#pullrequestreview-1867458208)_
- Should I do it?

Regards,
Th.

**Comment by The-Lum** [comment]
_[Just for traceability]_
That is now done with:
- #97

Thanks a lot. 👍

**Pull request merged**